intfsyncd/intfsync.cpp

```diff
@@ -46,6 +51,12 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
 
     key = LinkCache::getInstance().ifindexToName(rtnl_addr_get_ifindex(addr));
+    /* Get mtu from link */
+    name = key.c_str();
+    link = LinkCache::getInstance().getLinkByName(name);
+    if (link != NULL)
```

Comment:
when will this condition happen, by the time you get an interface, you should have the link?

Reply:
Yes. The router interface is attached to the link, by the time we create the router interface, we should have the link ready.